src/openrct2-ui/windows/Main.cpp

```diff
@@ -53,9 +53,14 @@ static Widget _mainWidgets[] = {
         }
 
     private:
-        void SetViewportFlags()
+        void SetViewportFlags(bool isTitleWindow)
         {
             viewport->flags |= VIEWPORT_FLAG_SOUND_ON;
+            if (isTitleWindow)
```

Comment:
Any particular reason over adding a parameter, as opposed to just testing for `gScreenFlags & SCREEN_FLAGS_TITLE_DEMO` here?

Reply:
No, just though it would make intent clearer, but I can change it.